### PR TITLE
[Term Entry] Lua Math .sin()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the Math.sin() term for Lua
- Closes Issue #3505 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/sin/sin.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+Title: 'sin()'
+Description: 'Returns the sine of an angle in radians.'
```
Comment (SSwiniarski) [inline content/lua/concepts/mathematical-library/terms/sin/sin.md:3]:
```suggestion
Description: 'Returns the sine of an angle given in radians.'
```

File: content/lua/concepts/mathematical-library/terms/sin/sin.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - 'Game Development'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`sin()`** function in Lua's math library calculates the sine of a given angle in radians. The result is the ratio of the length the side opposite the given angle to the length of the hypotenuse in a right-angled triangle.
```
Comment (SSwiniarski) [inline content/lua/concepts/mathematical-library/terms/sin/sin.md:15]:
```suggestion
The **`sin()`** function in Lua's math library calculates the sine of an angle given in radians. The result is the ratio of the length the side opposite the given angle to the length of the hypotenuse in a right-angled triangle.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Fejiro001 I did a first review and had two minor comments.

**Comment by Fejiro001** [comment]
@SSwiniarski ok thank you.

**Comment by SSwiniarski** [comment]
@Fejiro001 have you made the changes? I don't see them yet.

**Comment by Fejiro001** [comment]
@SSwiniarski I have made the changes now

**Review by SSwiniarski — APPROVED** [review]
@Fejiro001 Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@Fejiro001 @SSwiniarski Just made some minor language edits. Looks good to merge

**Comment by Fejiro001** [comment]
Thank you @SSwiniarski and @caupolicandiaz

**Pull request merged**